charts/aws-efs-csi-driver/values.yaml

```diff
@@ -66,6 +66,11 @@ controller:
     #   cpu: 100m
     #   memory: 128Mi
   nodeSelector: {}
+  updateStrategy: {}
+  # type: RollingUpdate
```

Comment:
Why would we add a comment with a strategy that is the [default](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) in Kubernetes? We should probably add a comment with good explanation. Something like: "override default strategy (RollingUpdate) to speed up large deployments" - and below add a comment/example of the strategy that provides better result at scale? Same for the node strategy below.

Reply:
@RomanBednar i added the comment you suggested. Not sure what to add as an example for large scale deployments , since that might depends of how many worker nodes you have and also what type of nodes (spot or on demand).